ios/Sources/MediaFeature/MediaListView.swift

```diff
@@ -0,0 +1,164 @@
+            if case let .searchText(text) = state.next,
+               !text.isEmpty {
+                isSearchResultVisible = true
+            } else {
+                isSearchResultVisible = false
+            }
+            if case .more = state.next {
+                isMoreActive = true
+            } else {
+                isMoreActive = false
+            }
+        }
+    }
+
+    enum ViewAction {
+        case moreDismissed
+    }
+
+    var body: some View {
+        ZStack {
+            ScrollView {
+                VStack(spacing: 0) {
+                    if viewStore.hasBlogs {
+                        MediaSection(
+                            icon: AssetImage.iconBlog.image.renderingMode(.template),
+                            title: L10n.MediaScreen.Section.Blog.title,
+                            store: store.scope(
+                                state: {  $0.list.blogs.map(\.feedItem) },
+                                action: { .init(action: $0, for: .blog) }
+                            )
+                        )
+                        divider
+                    }
+                    if viewStore.hasVideos {
+                        MediaSection(
+                            icon: AssetImage.iconVideo.image.renderingMode(.template),
+                            title: L10n.MediaScreen.Section.Video.title,
+                            store: store.scope(
+                                state: {  $0.list.videos.map(\.feedItem) },
+                                action: { .init(action: $0, for: .video) }
+                            )
+                        )
+                        divider
+                    }
+                    if viewStore.hasPodcasts {
+                        MediaSection(
+                            icon: AssetImage.iconPodcast.image.renderingMode(.template),
+                            title: L10n.MediaScreen.Section.Podcast.title,
+                            store: store.scope(
+                                state: {  $0.list.podcasts.map(\.feedItem) },
+                                action: { .init(action: $0, for: .podcast) }
+                            )
+                        )
+                    }
+                }
+            }
+            .zIndex(0)
+
+            if viewStore.isSearchResultVisible {
+                SearchResultView()
+                    .zIndex(1)
+            }
+        }
+        .background(
+            NavigationLink(
+                destination: IfLetStore(
+                    self.store.actionless.scope(state: MediaDetail.ViewState.init(state:)),
+                    then: MediaDetail.init(store:)
+                ),
+                isActive: viewStore.binding(
+                    get: \.isMoreActive,
+                    send: { _ in .moreDismissed }
+                )
+            ) {
+                EmptyView()
+            }
+        )
+    }
+
+    private var divider: some View {
```

Comment:
the divider in HomeScreen looks the same.
It seems better create common components or just add foregroundcolor.

Reply:
There are two kinds of dividers (0.5pt and 1pt).
I am confirming with the designer.